services/src/main/java/org/keycloak/services/managers/AuthenticationSessionManager.java

```diff
@@ -154,6 +158,29 @@ public void setAuthSessionCookie(String authSessionId, RealmModel realm) {
         log.debugf("Set AUTH_SESSION_ID cookie with value %s", encodedAuthSessionId);
     }
 
+    /**
+     * Sets the authentication session cookie with a value that combines both the user session id and root authentication
+     * session id, separated by a {@code '/'} character.
```

Comment:
I assume that such a cookies is not understood by KC19, so it wouldn't allow KC19 and KC20 to be run simultaneously. If we don't support zero-downtime from KC19 -> KC20 yet, this wouldn't matter. If we do, this should be mentioned in the release notes. 

Even if this is an internal behavior, could it still affect consumers, and should it therefore be mentioned in the release notes?

Reply:
That is actually a very good point - @hmlnarik what is your take on this?

Reply:
I was thinking - even if we come up a with a way to find the root auth session without changing the cookie - like changing the code that locates the auth session using the cookie to no longer assume the value should match a root session id - we would still be incompatible with KC19 because only KC20 would have smarter auth session location code. Once we've broken the assumption that user session id and root auth session id are the same, the compatibility is gone.

Reply:
I think someone who uses a zero-downtime-store (that is still in preview) could run a separate instance and ensure that traffic is separated between the two instances. So I'll only argue this should be in the release note, not that the code should change.